NRLMMD-GEOIPS/geoips
# 30 Make geotiffs right side up

# Reviewer Checklist

* [x] Required ***existing tests*** pass (ahi geotiff test)
* [x] NO required ***unit tests*** (unit tests coming soon!)
* [x] NO Required ***integration tests*** (tested with existing tests)

# Related Issues
fixes #30 

# Testing Instructions
To test this branch you can run ./tests/scripts/ahi.tc.WV.geotiff.sh, which will output 0. Also, the way I ensured the Geotiffs were now correct, was by running `conda install -c condo-forge gdal`, and once that was installed, you can run the command `gdalinfo <path_to_geotiff>` which will produce the following output.

<img width="1004" alt="Screenshot 2023-10-05 at 10 08 46 AM" src="https://github.com/NRLMMD-GEOIPS/geoips/assets/62626811/e8135a43-beb1-43f9-af31-6a5e626e5cd1">

As you can see above, the corner coordinates are correct for each corner, whereas in the previous version Upper Left was actually below Lower Left, causing the flip.

# Summary
This branch fixes the upside down geotiffs bug that was occurring when loading the .tif into other softwares, such as Google Earth Pro. To do this, the following files were modified.

  - geoips/geoips/plugins/modules/output_formatters/geotiff_standard.py
  - geoips/tests/outputs/ahi.tc.WV.geotiff/20200405_000000_SH252020_ahi_himawari-8_WV_100kts_100p00_1p0.tif

# Output
New output in Google Earth Pro

<img width="794" alt="Screenshot 2023-10-05 at 10 11 58 AM" src="https://github.com/NRLMMD-GEOIPS/geoips/assets/62626811/2176bf38-f4a3-4dbf-97c7-6e4139cf8d3c">
